### ISSUE TITLE
@clayui/css: Navigation Bar focus outline should be rounded

### ISSUE THREAD

**Issue by pat270** [liferay/clay#5437]
https://issues.liferay.com/browse/LPS-121102

<img width="960" alt="2021-01-08_19h45_50" src="https://user-images.githubusercontent.com/788266/227348863-9b4d4aec-2297-4482-ab25-de2a8f2e28d5.png">
